### DISCUSSION TITLE
Running Zettrl under Wayland

### DISCUSSION THREAD

**Discussion by vsanchez** [Zettlr/Zettlr#2741] [Show and tell]
Hi, to run Zettlr (and other electron apps) under wayland you must disable the GPU before trying to use the Ozone platform. If you do not do that, then the main window will not draw (the app appears to be running but no window is created). This problem was at least present in the KDE environment. I am using the latest version of the plasma system. This is the command:

Zettlr  --disable-gpu --enable-features=UseOzonePlatform --ozone-platform=wayland

This fixes the blurriness and the fact that the correct keymap was not applied. (A problem still present in Plasma for clients that use the XWayland system).

This is my system:

Operating System: Kubuntu 21.10
KDE Plasma Version: 5.23.2
KDE Frameworks Version: 5.87.0
Qt Version: 5.15.2
Kernel Version: 5.13.0-20-generic (64-bit)
Graphics Platform: Wayland
Processors: 8 × Intel® Core™ i7-8550U CPU @ 1.80GHz
Memory: 15.4 GiB of RAM
Graphics Processor: Mesa Intel® UHD Graphics 620

**Comment by lilithium-hydride** [comment]
This issue reproduces with Sway and Nvidia proprietary, and adding `--disable-gpu` does indeed fix it. Unfortunately, attempting to open the settings window spits out the following error:
```
[3196:0101/003701.508111:ERROR:bus.cc(393)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
```
Not a *massive* issue, since the config is in plain text and I don't often need to go there, but worth documenting nonetheless.
